.github/workflows/perform-release.yml

```diff
@@ -113,8 +113,8 @@ jobs:
 
       - name: "Deploy"
         run: |
-          MVN_ARGS="${{ env.MVN_CLI_ARGS }} deploy -Drelease -s settings.xml"
-          mvn $MVN_ARGS
+          MVN_ARGS="${{ env.MVN_CLI_ARGS }} -Drelease -s settings.xml"
+          mvn deploy $MVN_ARGS
```

Comment:
I tried using `mvn org.sonatype.central:central-publishing-maven-plugin:publish`, but that doesn't work on its own